### ISSUE TITLE
Load new content inside container div

### ISSUE THREAD

**Issue by Madebym** [blivesta/animsition#67]
I would like to animate new content into a container div, basically like doing it with AJAX.

Something like this, fixed navigation and logo on top, content container in the middle and footer at the bottom.

Navigation and footer are always present, only new content is animated using Animsition.

I tried a few solutions, but it always loads the whole page. Am I missing something obvious?

**Comment by anamostafa** [comment]
yes that wood be awesome, I am also interested with that . is in body has a solution ?

**Comment by johnbdot** [comment]
i actually managed to get it to work with ajax but i'm pretty new to javascript, so i will try my best to explain.

i was looking all over the website grabbing scripts from examples to try out and i just used the code that was found on the bottom of the main page, which is this:

var $animsition = $('.animsition');

$(document).ready(function() {
  $animsition.animsition({
    onLoadEvent: false
  })
  .each(function(i){
    $(this).animsition('in');
  });
});

$(document).on('click', '.foo', function(event){
  event.preventDefault();
  $animsition.animsition('out', $(this), $(this).attr('href'));
});

however, the trick is to put the code into the page that you're calling. for example: my index.html has a div with an id of <div id="ajax-content"> that is calling content from blog.html. all you have to do is put this code into the blog.html and NOT the index.html. this is because as the index.html is grabbing content from blog.html, the blog.html will cause the code to initialize as they load into <div id="ajax-content">.

now the only problem i am having is trying to get the 'out' animation to work, as the 'in' animation seems to be the only one working.

**Comment by marypo** [comment]
1+

**Comment by marypo** [comment]
Found a solution which eliminates the flashing of fixed page elements by using smoothState.js: http://blog.regencysoftware.com/post/2015/12/10/html-page-transitions-animiations

**Comment by ScottBeeson** [comment]
I would like this too.  I tried simply calling $('#mydiv').animsition('in'); method but it says "Cannot read property 'options' of undefined"
